tools/tertiary-analysis/scanpy/scanpy-normalise-data.xml

```diff
@@ -16,17 +16,30 @@ PYTHONIOENCODING=utf-8 scanpy-normalise-data.py
     #if $save_raw
```

Comment:
If I were to write this today (after seeing corrections by IUC), I would do
```
${save_raw}
```
instead of
```
#if $save_raw
    '${save_raw}'
#end if
```

Similarly for
```
#if $export_mtx
    '${export_mtx}' ./
#end if
```
I would replace with
```
${export_mtx}
```
and use `truevalue="--export-mtx ./"`.

However, these are just styles, perhaps doesn't matter that much.

Reply:
That is the first think I tried, but it gives an error because the whole argument gets passed as `'--export-mtx ./'` and it doesn't recognize it.

Reply:
I have set it up as you suggest now!